src/client/src/services/model/currencyPairPosition.js

```diff
@@ -1,10 +1,14 @@
+import CurrencyPair from './currencyPair';
+
 export default class CurrencyPairPosition {
   _symbol:string;
+  _currencyPair:string;
   _basePnl:number;
   _baseTradedAmount:number;
 
   constructor(symbol:string, basePnl:number, baseTradedAmount:number) {
     this._symbol = symbol;
+    this._currencyPair = new CurrencyPair(symbol);
```

Comment:
Can we just pass the currency pair via the ctor. The reference data service should have loaded it so we should pass that instance